compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -827,7 +827,15 @@ execGenerateSrc opts dalfOrDar mbOutDir = Command GenerateSrc Nothing effect
             stablePkgIds :: Set.Set LF.PackageId
             stablePkgIds = Set.fromList $ map LF.dalfPackageId $ MS.elems stableDalfPkgMap
 
-            genSrcs = generateSrcPkgFromLf pkgMap (getUnitId unitId unitIdMap) stablePkgIds (Just "CurrentSdk") pkg
+            config = DataDeps.Config
+                { configPackages = pkgMap
+                , configGetUnitId = getUnitId unitId unitIdMap
+                , configStablePackages = stablePkgIds
+                , configDependencyPackages = Set.empty -- this isn't right, is it?
```

Comment:
Yeah, that’s off. You should be able to get the dependencies from `GeneratePackageMap`. That said, this command is only for debugging (and tbh I don’t even use it for that) so I’m fine with addressing this separately.

Reply:
ok, I'm using those package ids for the dependencies here. :-)